src/wagtailtrans/wagtail_hooks.py

```diff
@@ -116,7 +115,7 @@ def edit_in_language_button(page, page_perms, is_parent=False):
     clear interface to work in.
 
     """
-    if not hasattr(page, 'language'):
+    if not issubclass(page.__class__, TranslatablePage):
```

Comment:
I'm sorry, I think my last comment was a bit misleading. I meant ``isinstance``, not sure why I said ``issubclass``!